edx_notifications/data.py

```diff
@@ -313,3 +313,36 @@ def id(self):
     # timestamps
     created = DateTimeField()
     modified = DateTimeField()
+
+
+class NotificationPreference(BaseDataObject):
+    """
+    Specifies the Notification preference.
+    """
+
+    # the internal name of the channel
+    name = StringField()
+
+    # display_name for notification preference
+    display_name = StringField()
+
+    # display description for notification preference
+    display_description = StringField()
+
```

Comment:
I just realized that we're going to need a 'default_value' field as well. Type: String()